grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/web/AppLifecycleBean.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.typedb.examples.fraud.web;
+
+import com.opencsv.bean.CsvToBean;
+import com.opencsv.bean.CsvToBeanBuilder;
+import com.typedb.examples.fraud.db.Dao;
+import com.typedb.examples.fraud.db.TransactionDao;
+import com.typedb.examples.fraud.model.Bank;
+import com.typedb.examples.fraud.model.BankCoordinates;
+import com.typedb.examples.fraud.model.Cardholder;
+import com.typedb.examples.fraud.model.Merchant;
+import com.typedb.examples.fraud.model.Transaction;
+import com.vaticle.typedb.client.api.TypeDBClient;
+import com.vaticle.typedb.client.api.TypeDBSession;
+import com.vaticle.typedb.client.api.TypeDBTransaction;
+import com.vaticle.typeql.lang.TypeQL;
+import io.quarkus.runtime.StartupEvent;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.stream.Collectors;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import org.jboss.logging.Logger;
+
+@ApplicationScoped
+public class AppLifecycleBean {
+
+  private static final Logger LOGGER = Logger.getLogger("AppLifecycleBean");
+
+  @Inject
+  TypeDBClient client;
+
+  @Inject
+  Dao<Bank> banks;
+  @Inject
+  Dao<Cardholder> cardholders;
+  @Inject
+  Dao<Merchant> merchants;
+  @Inject
+  TransactionDao transactions;
+
+  void onStart(@Observes StartupEvent ev) {
```

Comment:
Is this something Quarkus needs in order to run? It seems we don't ever use the `ev` parameter.

Reply:
Correct. Quarkus invokes this method.